openscap_daemon/cve_scanner/scan.py

```diff
@@ -26,12 +26,18 @@
 import StringIO
 from Atomic.mount import DockerMount
 from openscap_daemon.cve_scanner.scanner_error import ImageScannerClientError
+import bz2
 
 # TODO: External dep!
 import rpm
 
 
 class Scan(object):
+
+    RHEL_7_OVAL = "com.redhat.rhsa-RHEL7.xml.bz2"
+    RHEL_6_OVAL = "com.redhat.rhsa-RHEL6.xml.bz2"
+    RHEL_5_OVAL = "com.redhat.rhsa-RHEL5.xml.bz2"
```

Comment:
Perhaps we can construct this using the template string in oscap_docker package. Then it is universal and this sort of regression won't happen when this is changed in oscap_docker again.

using `getCVEInput.dist_cve_name`